### PR TITLE
Setup Configuration Environments

### PR DESCRIPTION
**Pull request by Huulamnguyen**
**What does this PR do?**
- This PR sets up Configuration Environment 

**Description of Task to be completed?**
- Setting up development, testing, and production configuration environments in config.py file

**How should this be manually tested?**
NAN

### CODE REVIEW

**Comment by alexkayabula** [comment]
@Huulamnguyen what kind of configuration are you setting up ?

**Comment by Huulamnguyen** [comment]
> @Huulamnguyen what kind of configuration are you setting up?

I just updated my description of the task. I wonder if I updated more detail about the configuration will answer your question??? Please let me know if you still have any concerns.

**Comment by alexkayabula** [comment]
@Huulamnguyen change the title to "Setup configuration environments".
Remove the description you have and give a brief statement.

File: config.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ENV = 'production'
+    SQLALCHEMY_DATABASE_URI = os.environ.get('PRODUCTION_DATABASE')
+
+config = {
+    'development': DevelopmentConfig,
+    'testing': TestingConfig,
+    'production': ProductionConfig,
+    'default': DevelopmentConfig
+} 
+
```
Comment (alexkayabula) [inline config.py:37]: @Huulamnguyen this code looks familiar. Are you reusing code from somewhere else ? If so you need to cite the source of the piece of code you are re-using. Please use this for reference [https://guides.libraries.uc.edu/citing/code](https://guides.libraries.uc.edu/citing/code).

Comment (Huulamnguyen) [reply config.py:37]: @alexkayabula Yes. I reused a source code. I will describe it clearly in my code base

**Comment by alexkayabula** [comment]
LGTM

**Pull request merged**